test/e2e/openshift.cnv.workflow.sh

```diff
@@ -12,15 +12,13 @@ export FIRST_SECONDARY_NIC=${FIRST_SECONDARY_NIC:-ens7}
 export SECOND_SECONDARY_NIC=${SECOND_SECONDARY_NIC:-ens8}
 export TIMEOUT=${TIMEOUT:-60m}
 
-if [ ! -f  $SSH ]; then
-    cat << EOF > ${SSH}
+cat << EOF > ${SSH}
```

Comment:
Please describe in the commit message why. I did not understood it from the PR description